src/Server/Server.php

```diff
@@ -168,8 +172,10 @@ public function getTokenCredentials(
     /**
      * Get user details by providing valid token credentials.
      *
-     * @throws CredentialsException If a "bad response" is received by the server
-     * @throws GuzzleException
+     * @param \League\OAuth1\Client\Credentials\TokenCredentials $tokenCredentials
```

Comment:
I think I see what's happening here. I'm very much against redundant docblocks. In fact, I might add this as a PHP-CS-Fixer rule. I only add them if they actually assist over what is provided at the language level. Could I be a pain and ask you to revert these? I'm 99% sure the previous state was correct, I went through every method and stripped redundant docblocks out.

Reply:
You're importing classes just for doc comments. This is not correct !

Reply:
I don't understand this last comment 😄 

The class `TokenCredentials` itself is already imported (so: no FQN) and the param has a FQN. Kind of "double redundant".

I agree with @bencorlett : throw out all phpdoc which is already crystal clear represented in the source 🏌️

Reply:
Yeah I’m not entirely following. 

There are classes where php storm adds an import statement to simplify a fqn in docblocks and I’m equally fine with this too. But that’s not what is the specific case here.

Reply:
How about `GuzzleException` ? it does exists in the code but it was imported !

Reply:
Check the usage of [GuzzleException ](https://github.com/thephpleague/oauth1-client/blob/master/src/Server/Server.php#L8)

Reply:
DocBlocks primarily serve as a blueprint for Doc generators. When you use aliases in them, generated Docs become unusable. As such, it is good practice (even the only correct path to take) to use FQNS in them. I agree with @arcanedev-maroc here.

Reply:
It seems to work fine for php scrutiniser, pho storm etc. in fact storm recommends it.

Reply:
I don’t use api doc generators for this project. These serve for IDE assistance.